libr/core/core.c

```diff
@@ -117,6 +117,42 @@ static void r_core_debug_syscall_hit(RCore *core) {
 	}
 }
 
+struct getreloc_t {
+        ut64 vaddr;
+        int size;
+};
+
+static int getreloc_tree(const void *user, const RBNode *n) {
+        struct getreloc_t *gr = (struct getreloc_t *)user;
+        const RBinReloc *r = container_of (n, const RBinReloc, vrb);
+        if ((r->vaddr >= gr->vaddr) && (r->vaddr < (gr->vaddr + gr->size))) {
+                return 0;
+        }
+
+        if (gr->vaddr > r->vaddr) {
+                return 1;
+        }
+        if (gr->vaddr < r->vaddr) {
+                return -1;
+        }
+        return 0;
+}
+
+// TODO: Use sdb in rbin to accelerate this
+// we shuold use aligned reloc addresses instead of iterating all of them
+R_API RBinReloc *r_core_getreloc(RCore *core, ut64 addr, int size) {
```

Comment:
this API should be in RBibn, there's nothing RCore specific in here

Reply:
Ok will move it to Rbin !